robot_info:
  num_robot: 6
  num_target: 2
  target_pos_flag: False